src/commands/skip.rs

```diff
@@ -0,0 +1,66 @@
+use std::sync::Arc;
+
+use anyhow::Result;
+use serenity::builder::CreateApplicationCommand;
+use serenity::{async_trait, model::prelude::command::CommandOptionType::Integer};
+
+use crate::command_context::CommandContext;
+use crate::store::guild_action::play_next_track::PlayNextTrack;
+use crate::store::guild_action::remove::Remove;
+
+use super::{CommandDataOptionUtil, JoovyCommand, JoovyCommands};
+
+const FROM: &str = "from";
+const TO: &str = "to";
+
+#[derive(Default, Debug)]
+pub struct Skip;
+
+#[async_trait]
+impl JoovyCommand for Skip {
+    fn create_application_command<'a>(
+        &self,
+        command: &'a mut CreateApplicationCommand,
+    ) -> &'a mut CreateApplicationCommand {
+        command
+            .name(JoovyCommands::Skip(Skip).as_ref())
+            .description("Removes tracks from the queue. If from is not defined the current track will be skipped.")
+            .create_option(|option| {
+                option
+                    .name(FROM)
+                    .kind(Integer)
+                    .description("Start number to remove from (inclusive)")
```

Comment:
"The track number to remove from" Idk om man behöver ha med inclusive, känns som det är standard för from när man integrerar med användare?

Reply:
Tycker det låter bättre. Vi kör på det!